Added text differentiating the dashboards (#1791)
wpilibsuite/frc-docs: source/docs/software/dashboards/smartdashboard/index.rst

SmartDashboard
==============

> SmartDashboard is a basic simple to use dashboard that uses relatively few computer resources.  It does not have the fancy look or some of the features Shuffleboard has, but it displays network tables data with a variety of widgets without bogging down the driver station computer.

Reviewer: ```suggestion
SmartDashboard is a simple and efficient dashboard that uses relatively few computer resources.  It does not have the fancy look or some of the features Shuffleboard has, but it displays network tables data with a variety of widgets without bogging down the driver station computer.
```
